src/nginxconfig/templates/global_sections/logging.vue

```diff
@@ -77,6 +77,90 @@ THE SOFTWARE.
                 </div>
             </div>
         </div>
+
+        <div class="field is-horizontal is-aligned-top">
+            <div class="field-label">
+                <label class="label">{{ i18n.templates.globalSections.logging.logformat }}</label>
+            </div>
+            <div class="field-body">
+                <div class="field">
+                    <div v-if="cloudflareEnabled" :class="`control${cloudflareChanged ? ' is-changed' : ''}`">
+                        <div class="checkbox">
+                            <PrettyCheck v-model="cloudflare" class="p-default p-curve p-fill p-icon">
+                                <i slot="extra" class="icon fas fa-check"></i>
+                                {{ i18n.templates.globalSections.logging.enableCloudflare }}
+                            </PrettyCheck>
+                        </div>
+                    </div>
+                    <div v-if="cfRayEnabled">
+                        <div v-if="cfConnectingIpEnabled" :class="`control${cfRayChanged ? ' is-changed' : ''}`">
```

Comment:
```suggestion
                        <div v-if="cfRayEnabled" :class="`control${cfRayChanged ? ' is-changed' : ''}`">
```
Extra div + incorrect if check?

Reply:
good catch, thanks!